app/main.py

```diff
@@ -10,5 +10,12 @@
 
 app = FastAPI()
 
+
+
 # Include routers
 app.include_router(locations.router, prefix="/api/v1", tags=["locations"])
+
+
+if __name__ == "__main__":
+    import uvicorn
+    uvicorn.run(app, host="127.0.0.1", port=8000)
```

Comment:
I have not setup my docker. So I was externally running uvicorn. If u want, I can comment this code and push it

I completely agree with Containerisation. I know a thing or two about it since I worked on the migration in my previous project.

Reply:
yep lets do that, if you are up for running as docker as well,.. I think the steps to run as docker container is present in the readme as well .. let me know if you are facing issues there.

Reply:
remove the part where we run this app directly with python and we should be good to go. we have the run command in the docker file